src/mudlet-lua/lua/LuaGlobal.lua

```diff
@@ -128,7 +128,8 @@ local packages = {
 -- TODO: extend to support common Lua code being placed in system shared directory
 -- tree as ought to happen for *nix install builds.
 local prefixes = {"../src/mudlet-lua/lua/", "../Resources/mudlet-lua/lua/",
-    "mudlet.app/Contents/Resources/mudlet-lua/lua/", "mudlet-lua/lua"}
+    "mudlet.app/Contents/Resources/mudlet-lua/lua/", "mudlet-lua/lua",
+    getMudletLuaDefaultPath()}
```

Comment:
@keneanung 
1. `getMudletLuaDefaultPaths` != `getMudletLuaDefaultPath`    
AND     
`getMudletLuaDefaultPath` != `getDefaultPaths` :stuck_out_tongue_winking_eye:  - but well spotted!

2. Does not the following code (lines 133-150 old; lines 134-151 new) handle stepping through the table?